app/views/proposals/details/_attachment.html.haml

```diff
@@ -16,3 +14,5 @@
           .medium-12.column
             = render partial: "proposals/details/attachment_list",
               locals: { proposal: @proposal }
+            - if !@proposal.completed? && !@proposal.canceled?
```

Comment:
Yep. He did it here: https://github.com/18F/C2/pull/1321

Before merging, can you make sure your branch is up to date? @micahsaul